src/hot-keys/RaceSpecificHotkeys/hotkeyData/human/buildings.ts

```diff
@@ -17,7 +17,8 @@ const buildings =
             Ability.Create('Train Peasant', 'btnpeasant', 'hpea', 'P', []),
             ...defaultMainBuilding(Ability.Default(), 'rhpm'),
             Ability.MultiHotkey('Upgrade to Keep/Castle', 'btncastle', 'hkee', 'U', [], ['hcas']),
-            Ability.Create('Call to Arms', 'btncalltoarms', 'amil', 'C', []),
+            Ability.UnhotkeyAbility('Call to Arms (For one or several peons)', 'btncalltoarms', 'amil', 'C', 'W'),
```

Comment:
maybe do this as Multihotkey aswell. So we have only one.

Reply:
Well in this case, there's 2 separate Call to Arms abilities: 1 for the Town Hall and 1 for the Peasants/Militia and they're both Unhotkey's. They'll still be one button. (I had this wrong initially, I had TWO Call to Arms in the Town Hall and I just commited a fix a few minutes ago for it)